packages/sync/src/modules/class-module.php

```diff
@@ -179,12 +179,31 @@ protected function count_actions( $action_names, $actions_to_count ) {
 	 * @access protected
 	 *
 	 * @param mixed $values Values to calculate checksum for.
+	 * @param bool  $sort If $values should have ksort called on it.
```

Comment:
Do we need to have this parameter? Do you see a place where we don't want to perform the sort, except to keep up with legacy sites that do not use the sorting?

Even if we force-sort the values, the worst case scenario would be that we will be syncing the callables for all those sites that upgrade and have issues with the sorted arrays. This should be a one-time deal and will keep the code complexity down.

Reply:
We really don't want there to be additional activity in the Activity Log or other downstream systems. To get around this we could potentially tie into the upgrade routine and have it go through and update all the checksums. However as that would be in the update routine itself we would still want to ensure that legit changes did get sent across.